app/views/companies/show.html.haml

```diff
@@ -3,4 +3,4 @@
     -actions = [link_to('Global Validity Check', "/rad_common/companies/#{Company.main.id}/global_validity_check", class: 'btn btn-default btn-sm', method: :post, data: { confirm: 'This is an exhaustive effort that could cause performance problems on the database. Are you sure?' })]
   = render 'layouts/panel_header', no_new_button: true, no_delete_button: true, no_index_button: true, additional_actions: actions
   .panel-body
-    = render 'layouts/show_data_columns', resource: @company, data: [{ label: 'Address', value: @company.full_address }, :phone_number, :website, :email]
+    = render 'layouts/show_data_columns', resource: @company, data: [{ label: 'Address', value: @company.full_address }, { label: 'Phone Number', value: (link_to @company.phone_number, "tel:#{@company.phone_number}") }, { label: 'Website', value: (link_to @company.website, "http://#{@company.website}", target: :_blank) }, { label: 'Email', value:(mail_to @company.email) }]
```

Comment:
we typically have the full web site including http so that would be duplicated, see [here](https://training.abetterwayinc.info/companies/1)